src/pages/RequisitionsPage.js

```diff
@@ -112,26 +126,51 @@ export class RequisitionsPage extends React.Component {
 
   renderCell(key, requisition) {
     switch (key) {
-      default:
-      case 'serialNumber':
-        return requisition.serialNumber;
       case 'entryDate':
         return requisition.entryDate.toDateString();
-      case 'numberOfItems':
-        return requisition.numberOfItems;
+      case 'otherStoreName':
+        return requisition.otherStoreName.name;
       case 'status':
         return formatStatus(requisition.status);
       case 'delete':
         return {
           type: 'checkable',
           icon: 'md-remove-circle',
-          isDisabled: requisition.isFinalised,
+          isDisabled: requisition.isFinalised || !requisition.isRequest,
```

Comment:
isRequest used 4 times, 2 directly and two with not, can make isResponse getter, if it's more readable